v2/azure-arm.yaml

```diff
@@ -869,64 +869,87 @@ status:
 objectModelConfiguration: 
   authorization:
     2020-08-01-preview:
+      RoleAssignment:
```

Comment:
If we're doing this, shouldn't the `include-transitive` section of this document be deleted? They aren't both needed, right?